flamedisx/xenon/x1t_sr1.py

```diff
@@ -200,6 +204,16 @@ class SR1Source:
     variable_elife = True
     path_electron_lifetimes = ('1t_maps/electron_lifetimes_sr1.json',)
 
+    # Comsol map
+    variable_drift_field = True
+    path_drift_field = 'nt_maps/fieldmap_2D_B2d75n_C2d75n_G0d3p_A4d9p_T0d9n_PMTs1d3n_FSR0d65p.json'
```

Comment:
nT map path for an 1T source, with a default `variable_drift_field = True`. To keep the 1T sources viable without config files you should switch off the default to `False`.

Reply:
I agree and I have just pushed the suggested change.